source/stylesheets/refills/_laptop.scss

```diff
@@ -0,0 +1,73 @@
+.laptop {
+  $lid-background: darken(white, 5);
+  $inset-shadow: inset 0 0px 1px transparentize(black, 0.8);
+  $laptop-size-ratio: 1.5;
+  $bottom-width: 22em * $laptop-size-ratio;
+  $laptop-lid-width: 17em * $laptop-size-ratio;
+  $screen-height: 137px * $laptop-size-ratio;
+  $radius-outer: 15px * $laptop-size-ratio;
+  $radius-bottom: 15px * $laptop-size-ratio;
+  $padding-screen: 15px * $laptop-size-ratio;
+  $bottom-height: 15px * $laptop-size-ratio;
+  $bottom-detail-width: 3em * $laptop-size-ratio;
+  $bottom-detail-height: 4px * $laptop-size-ratio;
+  $shadow-height: 4px * $laptop-size-ratio;
+  $shadow-width: 25em * $laptop-size-ratio; 
+  $shadow-distance: -1px * $laptop-size-ratio;
+  $outlines: 1px solid silver;
+
+  margin: auto;
+  position: relative;
+
+  .laptop-lid {
+    background: $lid-background;
+    border-radius: $radius-outer $radius-outer 0 0;
+    border: $outlines;
+    box-shadow: inset 0 2px 0 white;
+    margin: auto auto -1px;
+    padding: $padding-screen;
+    width: $laptop-lid-width;
+  }
+
+  .laptop-screen {
+    background: url('https://raw.githubusercontent.com/thoughtbot/refills/master/source/images/refills-screenshot2.jpg');
```

Comment:
Prefer double-quoted strings<br>URL `https://raw.githubusercontent.com/thoughtbot/refills/master/source/images/refills-screenshot2.jpg` should not contain protocol or domain